lab-david/README.md

```diff
@@ -0,0 +1,29 @@
+# Cowsay Http Server
```

Comment:
Great read me. The only thing I would maybe add is how to initially start the cowsay from the terminal. You basically want to describe the "what" and "how" of your app